copier/config/factory.py

```diff
@@ -0,0 +1,59 @@
+from typing import Tuple
+
+from ..types import AnyByStrDict, OptAnyByStrDict, OptBool, OptStrSeq
+from .objects import DEFAULT_DATA, ConfigData, EnvOps, Flags
+from .user_data import load_config_data, query_user_data
+
+__all__ = "make_config"
```

Comment:
This is invalid. `__all__` must be a list (any iterable actually), so
```python
__all__ = "make_config"
```
is like wrtiting 
```python
__all__ = ["m", "a", "k", "e", "_", "c", "", "o", "n", "f", "i", "g"]
```
This should be `__all__ = ["make_config"]`
See: https://docs.python.org/3/tutorial/modules.html#importing-from-a-package

Reply:
Thanks for your compliments. Your code was a joy to work with, especially due to the extensive test coverage provided.

Could `black` be the culprit here?
I do not remember having made this change manually.

Reply:
@jpscaletti
Could you maybe add some regression tests to ensure the integrity of the `__all__` dunders?